wheelsproxy/management/commands/sync_package.py

```diff
@@ -0,0 +1,12 @@
+import djclick as click
+from djclick.params import ModelInstance
+
+from ...models import BackingIndex
+
+
+@click.command()
+@click.argument('index', type=ModelInstance(BackingIndex, lookup='slug'))
+@click.argument('package', type=ModelInstance(Package, lookup='slug'))
```

Comment:
Using model instance here prevents syncing of non-existing packages. Just remove type and use ‘package’ directly in the call at line 11.